### PR TITLE
KIALI-840 Avoid sidebar to cover the service graph when resized

### PR DESCRIPTION
**Pull request by josejulio**
kiali-840

// cc: @mwringe

### CODE REVIEW

File: src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx [hunk 1]
```diff
@@ -93,14 +92,14 @@ export class CytoscapeReactWrapper extends React.Component<CytoscapeReactWrapper
     );
 
     this.cy = cytoscape(opts);
-    this.panzoom = this.cy.panzoom();
```
Comment (josejulio) [inline src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx]: @israel-hdez updated this code, it returns the `cy` instance and receives the init arguments or the `destroy` command:
see: https://github.com/cytoscape/cytoscape.js-panzoom/blob/master/cytoscape-panzoom.js#L561

File: src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx [hunk 2]
```diff
@@ -93,14 +92,14 @@ export class CytoscapeReactWrapper extends React.Component<CytoscapeReactWrapper
...
+    // Receives object with default init values or 'destroy' command and returns cy instance
+    this.cy.panzoom();
+    this.cy.container().querySelector('.cy-panzoom').style.zIndex = -1;
```
Comment (israel-hdez) [inline src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx]: do we want to change z-index?

Comment (josejulio) [reply src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx]: Yes, if we don’t, the panzoom will be over the left menu when scrolling and that looks wrong

Comment (israel-hdez) [reply src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx]: But the graph also goes on top of the controls :/
I'm ok with that if you see it right.

Comment (josejulio) [reply src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx]: Didn’t notice that. Will see if can workaround that.

**Comment by josejulio** [comment]
@israel-hdez updated the code, moved the z-index to other div

**Pull request merged**